static/styles.css

```diff
@@ -0,0 +1,46 @@
+body {
```

Comment:
This file is duplicated

Reply:
Thank you for pointing that out! I will remove the duplicate `styles.css` file and ensure that the correct one is in the `templates/static` directory.

I'll make the necessary changes and update the PR.